ml/src/main/scala/ai/h2o/sparkling/ml/features/H2OTargetEncoderBase.scala

```diff
@@ -0,0 +1,35 @@
+package ai.h2o.sparkling.ml.features
+
+import ai.h2o.sparkling.ml.params.H2OTargetEncoderParams
+import org.apache.spark.h2o.Frame
+import org.apache.spark.ml.PipelineStage
+import org.apache.spark.sql.types.{DoubleType, StructField, StructType}
+
+trait H2OTargetEncoderBase extends PipelineStage with H2OTargetEncoderParams {
+  override def transformSchema(schema: StructType): StructType = {
+    validateSchema(schema)
+    StructType(schema.fields ++ getOutputCols().map(StructField(_, DoubleType, nullable = true)))
```

Comment:
Are we sure output cols will have type of `DoubleType` ?

Reply:
TE returns a prior or posterior mean or a mixture of both.

https://github.com/h2oai/h2o-3/pull/3282/files#diff-2d747267b5dfe86c069723201c2cdd97R46

Reply:
Thanks!